verilog/analysis/checkers/token_stream_lint_rule_test.cc

```diff
@@ -41,6 +41,7 @@ TEST(StringLiteralConcatenationTest, FunctionFailures) {
        "\"Humpty Dumpty had a great fall.\"};", "\nendmodule"},
       {"module m;\n", "string tmp = {\"Humpty Dumpty \\ sat on a wall.\",",
        "\"Humpty Dumpty had a great fall.\"};", "\nendmodule"},
+      {"module a();\nstring x = 1 ? \"foo\" : \"bar\";\nendmodule"},
```

Comment:
Are there otherr situations where the previous code would've stumbled upon ? If you are aware of some, we should probably add them.

Also since we now know that `?:` is actually showing up as more complicated node-structure, we should test that we can detect line-continuation attempts in each branch (line continuation attempted in`"foo"` or in `"bar"` ?)

Reply:
I have split the test structure, and few more test cases were added, but just for now I have no more ideas